### PR TITLE
Minecraft 1.18

### PR DESCRIPTION
**Pull request by ModProg**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -3,14 +3,16 @@ org.gradle.jvmargs=-Xmx1G
 
 # Fabric Properties
 	# check these on https://fabricmc.net/versions.html
-	minecraft_version=1.17.1
-	yarn_mappings=1.17.1+build.14
-	loader_version=0.11.6
+	minecraft_version=1.18
+yarn_mappings=1.18+build.1
+loader_version=0.12.8
```
Comment (andrewlalis) [inline gradle.properties:8]: Fix indentation

**Review by andrewlalis — CHANGES_REQUESTED** [review]
If you can fix the problem with the source files appearing as new files under a `remappedSrc` directory and any other issues I commented on, then I can merge this.

**Comment by ModProg** [comment]
> If you can fix the problem with the source files appearing as new files under a `remappedSrc` directory and any other issues I commented on, then I can merge this.

oh yeah of course I must have missed those when committing :D

**Comment by andrewlalis** [comment]
Looks like the Github action for `gradle build` is failing because the workflow hasn't been updated to build with java 17. Additionally, `modid.mixins.json` still has a `compatibilityLevel` set to `JAVA_16`. Maybe that should be updated as well?

**Comment by ModProg** [comment]
> Additionally, `modid.mixins.json` still has a `compatibilityLevel` set to `JAVA_16`. Maybe that should be updated as well?

Does 1.18 only support 17?

**Comment by ModProg** [comment]
oh aparently they updated: https://www.pcgamesn.com/minecraft/1-18-pre-release-2

**Pull request merged**